spec/controllers/usuarios_controller_spec.rb

```diff
@@ -92,6 +92,26 @@
     end
   end
 
+  describe 'Solicitando recuperacao de senha' do
+    it 'sem informar e-mail' do 
+      expect do 
```

Comment:
esse teste não será executado do jeito que está. 
quando temos um expect do end ele espera que tenha um .to logo em seguida, pq vc está esperando algo pra comparar
nesse caso nao deveria ter esse expect do end

Reply:
O teste deveria ser assim:
describe 'Solicitando recuperacao de senha' do
    it 'sem informar e-mail' do
        post :recuperar_senha, params: {email:""}
        expect(response.body).to match 'E-mail não informado' 
      end

Reply:
é legal executar o simplecov localmente e consultar o index.html na pasta coverage pra ver os testes que estão sendo executados (linhas em verde)